static/js/shield_study/main.js

```diff
@@ -0,0 +1,15 @@
+
+function trackEvent(event) {
+    var action = (Math.random() >= 0.5);
+    _gaq.push(['_trackEvent', 'opt-in', 'click-' + action]); // Will need to filter on utm_source
+    event.target.innerHTML = "Thank you! Opted In";
+    event.target.setAttribute("onclick","");
+    if (action) {
+      window.location.href = "https://addons.mozilla.org/firefox/downloads/latest/x-screen-performance-study-1/";
```

Comment:
We wanted to treat this as an A/B test since the Shield addon will be able to install secondary addons without the scary addon installation prompt.  We wanted to see how this step of the funnel is affected by no message.

However, I'm currently trying to convince my coworkers that we don't need to structure this as an A/B test since the no-installation branch is the same as the installation branch less the last step.  Unless I'm missing something I'd like to delete this.  Will update shortly